lib/livebook_web/live/session_live/insert_buttons_component.ex

```diff
@@ -38,28 +38,64 @@ defmodule LivebookWeb.SessionLive.InsertButtonsComponent do
             </button>
           </:content>
         </.menu>
-        <%= if @smart_cell_definitions != [] do %>
-          <.menu id={"#{@id}-smart-menu"} position="left">
-            <:toggle>
-              <button class="button-base button-small">+ Smart</button>
-            </:toggle>
-            <:content>
-              <%= for smart_cell_definition <- Enum.sort_by(@smart_cell_definitions, & &1.name) do %>
-                <button class="menu-item text-gray-500"
-                  role="menuitem"
-                  phx-click="insert_cell_below"
-                  phx-value-type="smart"
-                  phx-value-kind={smart_cell_definition.kind}
-                  phx-value-section_id={@section_id}
-                  phx-value-cell_id={@cell_id}>
-                  <span class="font-medium"><%= smart_cell_definition.name %></span>
-                </button>
-              <% end %>
-            </:content>
-          </.menu>
+        <%= cond do %>
+          <% @runtime == nil -> %>
+            <span class="tooltip right" data-tooltip={"Please start a runtime (or run any cell)\nto see the available smart cells"}>
+              <button class="button-base button-small" disabled>+ Smart</button>
+            </span>
```

Comment:
@josevalim we discussed using a modal for this initial case too, but I'm thinking it may be more intuitive/educative to tell the user what and how to do instead. If you have stronger preference for the modal, it's also fine by me :)

Reply:
I read that disabled buttons are bad for UI in general, because it is not immediately clear why they are disabled (i.e. it requires mousing over, which is not straightforward in some devices or some workflows). So I think we should go with the modal?